code/game/objects/items.dm

```diff
@@ -306,6 +306,37 @@
 /obj/item/proc/on_found(mob/wearer, mob/finder)
 	return
 
+// have your item's MouseDropFrom call this if mouse-dropping is the default way of picking up your item, such as a paper bin or a deck of card
+/obj/item/proc/MouseDropPickUp(atom/over_object)
+	var/mob/user = usr
+	if(!ishigherbeing(user) || user.incapacitated())
```

Comment:
do robots count as higher being?

Reply:
No. Higher beings are
* Humans
* Martians
* Monkeys with dexterity

Reply:
Is it an issue?

Reply:
I was hoping jkn would answer this since he brought it up. I can only see an issue for MoMMIs trying to move paper bins and such.

Reply:
alright I added an exception for robots